testing/test_warnings.py

```diff
@@ -424,6 +424,26 @@ def test():
     result.stdout.fnmatch_lines(["* 1 failed in*"])
 
 
+def test_accept_unknown_category(pytester: Pytester) -> None:
+    """Filters defined by marks should always take precedence (#3946)."""
```

Comment:
#3946 is related, but I think it is better to use this PR for future reference.

```suggestion
    """Filters defined by marks should always take precedence (#13732)."""
```